src/@next/IndexTable/index.ts

```diff
@@ -0,0 +1 @@
+export * as IndexTable from './IndexTable';
```

Comment:
missing export in `src/@next/index`?

Reply:
my bad. i removed the `as` and also added the export